week-2/E-conditionals/exercise.js

```diff
@@ -3,19 +3,12 @@
   ---------------------------------
   Add an if statement to only show the name if the role is student
 */
-
-
-var name = "Daniel";
-var role = "mentor";
-
 var name2 = "Harun";
 var role2 = "student";
 
-
-console.log("Hi, My name is " + name);
-console.log("Hi, My name is " + name2);
-
-
+if (role2) {
```

Comment:
This is saying, 'if role2 exists, console.log the below string'. Instead, you want to ask whether role2 === 'student', right?